app/frontend/streamlit_app.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+'''
+Talk2BioModels: Interactive BioModel Simulation Tool
+'''
+
+import os
+import sys
+import streamlit as st
+sys.path.append('./')
+from agents.talk2biomodels.tools.ask_question import AskQuestionTool
+from agents.talk2biomodels.tools.simulate_model import SimulateModelTool
+from agents.talk2biomodels.tools.plot_figure import PlotImageTool
+from agents.talk2biomodels.tools.model_description import ModelDescriptionTool
+from langchain_core.tools import tool
+from langchain.agents import create_tool_calling_agent, AgentExecutor
+from langchain_openai import ChatOpenAI
+from langchain_core.prompts import ChatPromptTemplate
+
+st.set_page_config(page_title="Talk2BioModels", page_icon="🤖", layout="wide")
```

Comment:
This is definitely fine for the moment. Any thoughts on how we can make the UI compatible with the different agents?